### ISSUE TITLE
Helper to prevent executing mode's body if there is data in bemjson

### ISSUE THREAD

**Issue by zxqfox** [bem/bem-xjst#245]
Related #175 

I believe we should have helpers as replacement for modes' default content.

`block('block2').match(function() { return !this.ctx.content; }).content()(() => 'default content');`.

`match(function() { return !this.ctx.content; })` can be something like `default` (but `def` already there). Can you think about better name?

The same can be useful for other modes:
```js
.match(function() { return !this.ctx.content; }).content()(() => 'default content');
.match(function() { return !this.ctx.tag; }).tag()(() => 'span');
```
etc.

**Comment by miripiruni** [comment]
we need statistic for such cases in real codebase

**Comment by miripiruni** [comment]
Related: #336

**Comment by miripiruni** [comment]
Closed in favor of #336
